recipes/libdc1394/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+
+class Libdc1394Conan(ConanFile):
+    name = "libdc1394"
+    version = "2.2.6"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = 'https://damien.douxchamps.net/ieee1394/libdc1394/'
+    description = "libdc1394 provides a complete high level API to control IEEE 1394 based cameras"
+    topics = ("conan", "ieee1394", "camera", "iidc", "dcam")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    _env_build = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_autotools(self):
+        if not self._env_build:
+            self._env_build = AutoToolsBuildEnvironment(self)
+            if self.options.shared:
```

Comment:
The configure script looks for libusb, sdl and opengl.
Perhaps optionally add the libusb requirement and disable SDL2 to have a reproducible build?
The configure script has also a ´--disable-examples´ option.

Reply:
added `--disable-examples`